qiskit_experiments/characterization/t1_analysis.py

```diff
@@ -27,20 +27,29 @@
 
 
 class T1Analysis(BaseAnalysis):
-    """T1 Experiment result analysis class.
-
-    Analysis Options:
-
-        * t1_guess (float): Optional, an initial guess of T1.
-        * amplitude_guess (float): Optional, an initial guess of the
-                                   coefficient of the exponent.
-        * offset_guess (float): Optional, an initial guess of the offset.
-        * t1_bounds (list of two floats): Optional, lower bound and upper
-                                          bound to T1.
-        * amplitude_bounds (list of two floats): Optional, lower bound and upper
-                                                 bound to the amplitude.
-        * offset_bounds (list of two floats): Optional, lower bound and
-                                              upper bound to the offset.
+    r"""A class to analyze T1 experiments.
+
+    Fit Model
+        The fit is based on the following decay function.
+
+        .. math::
+
+            F(x) = a e^{-x/t1} + b
```

Comment:
In the Tutorial the decay function is written as F(t) = A e ^ {-t/T1} + C. For consistency, I would change this according to the tutorial

Reply:
I'll replace C by B in the tutorial, to make it closer to the curve analysis documentation